tests/smoke-go-update-security-multidir.yaml

```diff
@@ -110,79 +116,15 @@ output:
             dependency_files:
                 - /go/multi-dir/foo/go.mod
                 - /go/multi-dir/bar/go.mod
-    - type: create_pull_request
```

Comment:
Not sure what was going on here before but the job definition was invalid: a `dependency-group-to-refresh` with no `dependency-groups` doesn't make sense.